src/clang_parser.cpp

```diff
@@ -230,8 +230,25 @@ void ClangParser::parse(ast::Program *program, BPFtrace &bpftrace, std::vector<s
       &translation_unit);
   if (error)
   {
-    std::cerr << "Clang error while parsing C definitions: " << error << std::endl;
-    std::cerr << "Input (" << input.size() << "): " << input << std::endl;
+    if (bt_debug == DebugLevel::kFullDebug) {
```

Comment:
Won't this mean we just have silent errors unless running in debug mode?

Reply:
We still get warnings and errors from clang, which is more useful/actionable for the user than this information.

Reply:
Also, I don't think I ever hit this branch, even if the clang parsing fails (which is why we check for clang diagnostics below).

Reply:
If I'm remembering right, this is the error we see in the LLVM5 static builds. If you don't see it in any other case we might as well keep it in non-debug mode

Reply:
oh skipped your first comment here - yeah guess it's fine to keep this for debug mode then